### DISCUSSION TITLE
No v0.4.24 wheels for cuda 11/12 in release repository?

### DISCUSSION THREAD

**Discussion by adam-hartshorne** [jax-ml/jax#19714] [General] [answered]
Despite the github saying there has been a 0.4.24 jaxlib and jax release, looking in https://storage.googleapis.com/jax-releases/jax_cuda_releases.html there doesn't seem to be such releases.

**Comment by yashk2810** [comment, marked as answer]
It's now fixed! Thanks for the report! We appreciate it :)
